environment/sql/data.sql

```diff
@@ -71,11 +71,13 @@ INSERT INTO `localisation` (`language`, `name`, `value`) VALUES
 ('en', 'COMMAND_JOIN', 'You need to be in a voice channel to use this command'),
 ('en', 'COMMAND_LOOPQUEUE', 'Loop queue %s'),
 ('en', 'COMMAND_VOLUME', 'Volume set to %d'),
+('en', 'COMMAND_VOLUME_ERROR', 'Volume can only set to a number between 0 and 100'),
+('en', 'COMMAND_VOLUME_MUTE', 'Bot is now muted'),
```

Comment:
Here I would rather write "Volume is now muted" or only "Volume muted". 
I think in other messages Hera usually refers to her self as "I".

Reply:
So "I am now muted" instead since you cant mute a volume

Reply:
Pretty sure you can say "Volume muted" or the likes, but yeah "I am now muted" is fine with me.